src/components/DataInput/DatePicker.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { DesktopDatePicker } from "@mui/x-date-pickers";
+
+const StyledDatePicker = styled(DesktopDatePicker)`
+    & .MuiInputBase-root {
+        & .MuiInputBase-input {
+            // pushes calendar icon to the right
+            width: 10000%;
```

Comment:
how does this work? (tried commenting out this line but doesn't seem to change anything on the form - might be looking at the wrong place tho)

Reply:
Well it used to move the icon to the right but it no longer does this I have absolutely zero clue here (we love MUI)

Reply:
:)